index.html

```diff
@@ -0,0 +1,234 @@
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <section class="contact">
+      <div class="contact_wrap">
+        <h1 class="head">Contact Form</h1>
+        <h3 class="bodyText">If you have an application you are interested in developing,
+          a feature that you need built or a project that needs coding. I’d love to help with it. </h3>
+        <form action="https://formspree.io/f/mzbyvzqb" method="post" class="contact_form">
+          <input type="text" required maxlength="30" placeholder="Alexandra Loremipsum" name="name" id="name" class="form_fields">
+          <input type="email" name="email" required id="email" placeholder="aoremipsum@mail.com" class="form_fields">
+          <textarea class="form_fields textarea" required maxlength="500" name='message' id='message' placeholder="Write your message here" cols="40"></textarea>
+          <input type="submit" value="Get In Touch" class="form_submit">
+        </form>
```

Comment:
- [ ] The [design given on figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) has a different type of design than the one you implemented here. 
- The design looks like this
![image](https://user-images.githubusercontent.com/64360974/123536772-f4302800-d734-11eb-95d7-549899627138.png)
- As you can see the given design and the one you worked on are not similar. I think its best to follow the design specified so kindly work on the contact form's design including font, colors, images, margins.